package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "todoapp",
+  "version": "1.0.0",
+  "description": "A simple TODO app",
+  "main": "index.js",
+  "scripts": {
+    "watch": "webpack --watch --mode development",
```

Comment:
Maybe it makes more sense to change this script name to dev since its purpose it's for development.